wtxmgr/tx.go

```diff
@@ -805,6 +805,95 @@ func (s *Store) rollback(ns walletdb.ReadWriteBucket, height int32) error {
 	return putMinedBalance(ns, minedBalance)
 }
 
+// OutputsToWatch returns a list of outputs to monitor during the wallet's
+// startup. The returned items are similar to UnspentOutputs, exccept the
+// locked outputs and unmined credits are also returned here. In addition, we
+// only set the field `OutPoint` and `PkScript` for the `Credit`, as these are
+// the only fields used during the rescan.
+func (s *Store) OutputsToWatch(ns walletdb.ReadBucket) ([]Credit, error) {
+	var (
+		unspent []Credit
+		op      wire.OutPoint
+		block   Block
+	)
+
+	err := ns.NestedReadBucket(bucketUnspent).ForEach(func(k,
```

Comment:
Will this panic if `bucketUnspent` hasn't been created yet?

Reply:
Think it'd always be created at this point.